pkg/skaffold/yamltags/tags.go

```diff
@@ -77,39 +74,42 @@ func GetYamlTag(value interface{}) string {
 	return rawStr[:i]
 }
 
-// GetYamlTags returns the tags of the non-nested fields of the given non-nil value
-// If value interface{} is
-// latest.DeployType{HelmDeploy: &HelmDeploy{...}, KustomizeDeploy: &KustomizeDeploy{...}}
-// then this parses that interface as it's raw yaml:
-// 	kubectl:
-//    manifests:
-//    - k8s/*.yaml
-//  kustomize:
-//    paths:
-//    - k8s/
-// and returns ["kubectl", "kustomize"]
-// empty structs (e.g. latest.DeployType{}) when parsed look like "{}"" and so this function returns []
-func GetYamlTags(value interface{}) []string {
+// GetYamlTags returns the first `yaml` tag value for each non-nested field of the given non-nil config parameter
+// For example if config is `latest.DeployType{HelmDeploy: &HelmDeploy{...}, KustomizeDeploy: &KustomizeDeploy{...}}`
+// then it returns `["kubectl", "kustomize"]`
+func GetYamlTags(config interface{}) []string {
```

Comment:
This isn't a tag, but a key (in YAML terms).  `GetYamlKeys()`?

Reply:
done

Reply:
done